tests/ReflectionClosure1Test.php

```diff
@@ -276,9 +276,9 @@
 
 test('interpolation1', function () {
```

Comment:
I think we want to keep this test, but just skip on php 8.2?

Reply:
Tried finding a way to check the tag bug `PHP_VERSION_ID` isn't working. Feel free to propose something 👍